src/Features/Core/Portable/RQName/ParenthesesTreeWriter.cs

```diff
@@ -1,19 +1,20 @@
 // Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
-using System;
-using System.Collections.Generic;
-using System.Linq;
 using System.Text;
-using System.Threading.Tasks;
-using Microsoft.VisualStudio.LanguageServices.Implementation.RQName.SimpleTree;
+using Microsoft.CodeAnalysis.Features.RQName.SimpleTree;
 
-namespace Microsoft.VisualStudio.LanguageServices.Implementation.RQName
+namespace Microsoft.CodeAnalysis.Features.RQName
 {
     internal static class ParenthesesTreeWriter
     {
         public static string ToParenthesesFormat(SimpleTreeNode tree)
         {
-            StringBuilder sb = new StringBuilder();
+            if (tree == null)
```

Comment:
:question: Why are we now allowing this case?

**Edit:** ➡️ This is an obfuscated null check for `RQNameInternal.From`.